adapters/faraday/gemfiles/faraday_0.13.gemfile

```diff
@@ -0,0 +1,12 @@
+# This file was generated by Appraisal
```

Comment:
I haven't used Appraisal before. Is it common practice to commit these generated files to source control?

Reply:
I'm not sure if it's common practice, but the Appraisal docs recommend it: https://github.com/thoughtbot/appraisal#version-control (commit gemfiles, exclude lock files).

Reply:
You _could_ probably just run `bundle exec rake appraisal generate` before running tests, but this is probably a shortcut.